exporter/datadogexporter/internal/logs/sender.go

```diff
@@ -71,8 +70,8 @@ func (s *Sender) SubmitLogs(ctx context.Context, payload []datadogV2.HTTPLogItem
 	// Correctly sets apiSubmitLogRequest ddtags field based on tags from translator Transform method
 	if payload[0].HasDdtags() {
 		tags := datadog.PtrString(payload[0].GetDdtags())
-		if s.opts.Ddtags != nil {
-			tags = datadog.PtrString(fmt.Sprint(*s.opts.Ddtags, ",", payload[0].GetDdtags()))
+		if s.verbose {
+			s.logger.Debug("Tags", zap.String("tags", *tags))
```

Comment:
We want to keep this code here in order to append tags to `s.opts.Ddtags` in the case that this field is set.
```suggestion
		if s.opts.Ddtags != nil {
			tags = datadog.PtrString(fmt.Sprint(*s.opts.Ddtags, ",", payload[0].GetDdtags()))
```